.babelrc

```diff
@@ -1,13 +1,4 @@
 {
-    "presets": ["es2015", "react"],
-    "env": {
-        "development": {
-            "presets": ["es2015", "react"],
-            "plugins": [
-                ["transform-es2015-modules-commonjs-simple", {
-                    "noMangle": true
-                }]
-            ]
-        }
-    }
-}
+    "presets": ["env", "react"],
+    "plugins": ["transform-object-rest-spread"]
```

Comment:
I think babel-preset-env includes transform-object-rest-spread at this point (it's stage 4)

Using "env" without setting explicit browser versions should result in this configuration being used:
> 0.5%, last 2 versions, Firefox ESR, not dead

Is this what we want/need?

Reply:
Hmm the repo even points out that it's now included in the main Babel repo: https://github.com/babel/babel-preset-env

Reply:
Although that's Babel `7.0`, which is a whole different beast :)